.github/workflows/package.yml

```diff
@@ -283,16 +283,16 @@ jobs:
           find . -name example-app -type d -exec rm -r {} +
           find . -name cmake -type d -exec rm -r {} +
           find . -name pkgconfig -type d -exec rm -r {} +
-          mv ${COMBINED_PACKAGE_DIR}/include/pcapplusplus/* ${COMBINED_PACKAGE_DIR}/include/
-          rmdir ${COMBINED_PACKAGE_DIR}/include/pcapplusplus/
-          mkdir -p android-package
-          mv ${COMBINED_PACKAGE_DIR} android-package
+          mv ${COMBINED_PACKAGE_DIR}/include/pcapplusplus/* "${COMBINED_PACKAGE_DIR}/include/"
+          rmdir "${COMBINED_PACKAGE_DIR}/include/pcapplusplus/"
+          mkdir -p "android-package"
+          mv "${COMBINED_PACKAGE_DIR}" "android-package"
 
       - uses: actions/upload-artifact@v4
         with:
-          name: android-package
-          if-no-files-found: error
           path: android-package
+          name: android-package-${{ matrix.target }}-${{ matrix.api-version }}
+          if-no-files-found: error
```

Comment:
This is a breaking change because until now we provided all archs in the same package. This is also the way we recommend integrating PcapPlusPlus in our example: https://github.com/seladb/ToyVpn-PcapPlusPlus

Maybe we can use `@actions/upload-artifact/merge` to merge the packages after they have been created?
https://github.com/actions/upload-artifact/blob/main/merge/README.md

Reply:
This is still the case we will merge and reupload année artifact combined

Reply:
Oh, this is what `merge-multiple` does?

Reply:
This allow to download multiple artifacts and to merge them in the same folder.

Then we repack it.

Reply:
ok sure 👍